app/helpers/admin/sidebar_helper.rb

```diff
@@ -44,7 +44,7 @@ def sidebar_tabs(tabs, options = {})
                          text
                        end
                      end
-      link = tag.a(link_content, "href" => "##{id}", "data-toggle" => "tab")
+      link = tag.a(link_content, href: "##{id}", "data-toggle": "tab")
```

Comment:
One thing I'd like to confirm before approving - if we run `rubocop -a`, do these change back to the 'faulty' version?

Reply:
It's OK, they're stable. `rubocop -a` will only correct code that's causing an issue. Since we're now using the prescribed `tag.x` method, there's no issue to correct.

Reply:
Thought so. Thanks 👍